docs/addons/argo-cd.md

```diff
@@ -1,22 +1,14 @@
-# Argo CD Add-on
+# Argo CD add-on
 
-[Argo CD](https://argoproj.github.io/argo-cd/) is a declarative, GitOps continuous delivery tool for Kubernetes. The Argo CD add-on provisions [Argo CD](https://argoproj.github.io/argo-cd/) into an EKS cluster, and can optionally bootstrap your workloads from public and private Git repositories. 
-
```

Comment:
Keep all of these docs. Fine to edit but we need the content.

Reply:
This is still intact.

Reply:
Sections were removed because the info is already documented in other locations.